encoder/vaapiencoder_base.cpp

```diff
@@ -89,12 +89,12 @@ YamiStatus VaapiEncoderBase::start(void)
 void VaapiEncoderBase::flush(void)
 {
     AutoLock l(m_lock);
```

Comment:
Hi @chivakker 
Agree with Lizhong, we should focus on h264, h265 in this patch. I think we had better not change every codec for this base on following reason:
1.	Currently, we do not have any thing in VaapiEncoderBase::flush, it do not mean we will not add something in future. 
2.	Very codec have right to choose call base class flush or not.
3.	If it is no functional or readability benefit. We had better not change the code. If we change vp9encoder, author need do test for vp9encoder. This is an extra effort for author and it's not introduce benefit 
Do you think it is make sense?